src/main/java/muramasa/antimatter/network/packets/CapabilityPacket.java

```diff
@@ -0,0 +1,51 @@
+package muramasa.antimatter.network.packets;
+
+import muramasa.antimatter.Antimatter;
+import muramasa.antimatter.capability.ICapabilityHost;
+import muramasa.antimatter.util.Utils;
+import net.minecraft.network.PacketBuffer;
+import net.minecraft.tileentity.TileEntity;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.world.dimension.DimensionType;
+import net.minecraft.world.server.ServerWorld;
+import net.minecraftforge.fml.network.NetworkEvent;
+import net.minecraftforge.fml.server.ServerLifecycleHooks;
+
+import java.util.function.Supplier;
+
+public class CapabilityPacket {
```

Comment:
Refer to prior comment.

Reply:
Also, i think that some of caps may be not needed to be send. But I am sure that covers, energy, interact (pipes) is needed. I can try to moved package sending from the class to individual handlers. I wil think about it tommorow then. I have another idea how to improve pr a bit